src/main/kotlin/net/maizegenetics/phgv2/pathing/AlignmentUtils.kt

```diff
@@ -17,6 +17,8 @@ import java.util.*
 import kotlin.math.ceil
 import kotlin.math.max
 import kotlin.math.min
+import kotlin.time.DurationUnit
```

Comment:
Is something being timed?

Reply:
I'm not sure where that came from.  I'll remove it.